Modules/timemodule.c

```diff
@@ -2053,7 +2053,11 @@ pysleep(_PyTime_t secs)
 {
     _PyTime_t deadline, monotonic;
 #ifndef MS_WINDOWS
+#ifdef HAVE_CLOCK_NANOSLEEP
+    struct timespec timeout;
```

Comment:
Usually, a timeout is relative. This one is special, it's absolute. Can you please rename it to "timeout_abs"?

Reply:
Done